lib/turn.rb

```diff
@@ -0,0 +1,27 @@
+class Turn
+    attr_reader :card, :guess, :feedback
+
+def initialize(guess, card)
+  @guess = guess
+  @card = card
+  @feedback = ''
+end
+
+def correct?
+  if @guess == card.answer 
+    true
+  else
+    false
+  end
+end
+
+def feedback
+  if correct? == true
+    @feedback = 'Correct!'
+  else
+    @feedback = 'Incorrect.'
+  end
+  @feedback
+end
```

Comment:
Going with the suggestion above, I'd suggest removing the attrreader AND the attribute `@feedback`, and just let this method return a string: 

```
def feedback
  if correct? == true
    return 'Correct!'
  else
    return 'Incorrect.'
  end
end
```

You're storing unnecessary state by having an attribute for feedback.